package.json

```diff
@@ -54,9 +59,11 @@
     "@pixi/filter-glow": "^4.1.5",
     "@tmcw/togeojson": "^5.1.2",
     "@turf/bbox-clip": "^6.0.0",
+    "benchmark": "^2.1.4",
```

Comment:
This should usually go under `devDependencies` I think - it's needed for the build/test but not meant to be included in the code.